### ISSUE TITLE
app crashes on error report via github

### ISSUE THREAD

**Issue by MathJoDE** [TeamNewPipe/NewPipe#5308]
### Checklist

- [x] I am using the latest version - 0.20.8
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug

1. raise any error (in my case it was #5306 ), so that you get an error page
2. click the "report on github" button at the bottom
3. confirm the privacy popup with "Accept"

### Actual behaviour
The app crashes (see screen recording below).

### Expected behavior
I don't know exacly, but maybe a browser window opening...

### Screenshots/Screen recordings

https://user-images.githubusercontent.com/59893632/103280051-af1e7980-49cf-11eb-91c1-4e883aee7eda.mp4

## Exception
* __User Action:__ ui error
* __Request:__ App crash, UI failure
* __Content Country:__ DE
* __Content Language:__ de-DE
* __App Language:__ en
* __Service:__ none
* __Version:__ 0.20.8
* __OS:__ Linux Android 7.0 - 24
<details><summary><b>Crash log </b></summary><p>

```
android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.VIEW dat=https://github.com/... flg=0x10000000 pkg=com.huawei.android.internal.app }
	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:1854)
	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1544)
	at android.app.Activity.startActivityForResult(Activity.java:4391)
	at androidx.fragment.app.FragmentActivity.startActivityForResult(FragmentActivity.java:675)
	at android.app.Activity.startActivityForResult(Activity.java:4335)
	at androidx.fragment.app.FragmentActivity.startActivityForResult(FragmentActivity.java:662)
	at android.app.Activity.startActivity(Activity.java:4702)
	at android.app.Activity.startActivity(Activity.java:4670)
	at org.schabi.newpipe.util.ShareUtils.openUrlInBrowser(ShareUtils.java:39)
	at org.schabi.newpipe.report.ErrorActivity.lambda$openPrivacyPolicyDialog$6(ErrorActivity.java:279)
	at org.schabi.newpipe.report.ErrorActivity.lambda$openPrivacyPolicyDialog$6$ErrorActivity(ErrorActivity.java)
	at org.schabi.newpipe.report.-$$Lambda$ErrorActivity$zOFMu5zpmIgbungN7tR4m3YS3ps.onClick(lambda)
	at com.android.internal.app.AlertController$ButtonHandler.handleMessage(AlertController.java:166)
	at android.os.Handler.dispatchMessage(Handler.java:105)
	at android.os.Looper.loop(Looper.java:156)
	at android.app.ActivityThread.main(ActivityThread.java:6523)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:942)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:832)

```
</details>
<hr>

### Device Model
Huawei P9 lite (HWVNS-L31)

[edit: device model information added]

**Comment by TiA4f8R** [comment]
Duplicate of #3925 (I made a PR with a fix: #5187 in which you can test a debug APK).

**Comment by MathJoDE** [comment]
Oh, I'm sorry!
